message-queue-consumer.gemspec

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+lib = File.expand_path('../lib/', __FILE__)
+$:.unshift lib unless $:.include?(lib)
+
+require 'message_queue_consumer/version'
+
+Gem::Specification.new do |s|
+  s.name = "message-queue-consumer"
+  s.version = MessageQueueConsumer::VERSION
+  s.authors = ["Stuart Gale"]
```

Comment:
I think this should be
`  gem.authors       = ["GOV.UK Dev"]`
(similar to govuk-admin-template).
We don't have a consistent pattern here, but I don't think individual developer names are helpful here.